command/jsonplan/plan.go

```diff
@@ -76,6 +77,14 @@ type output struct {
 	Value     json.RawMessage `json:"value,omitempty"`
 }
 
+// variables is the JSON representation of the variables provided to the current
+// plan.
+type variables map[string]interface{}
```

Comment:
Given how this is used below, I think this could be `map[string]*variable`, right?